docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -3712,6 +3716,60 @@ ATTRIBUTES ALIGNED FORMAT
 --- SQL operation complete.
 ```
 
+* This example demonstrates the character set of NCHAR column (_c1_) *cannot be specified*.

```

Comment:
I think one example is probably enough. The point we are trying to make is that "NATIONAL CHAR" implies a particular character set, namely, "UCS2". This is hard-coded into the product; it cannot be changed. Possible word-smith: "NATIONAL CHAR (or NCHAR) is a short-hand for CHAR ... CHARACTER SET UCS2. So it is not possible to specify a CHARACTER SET clause when using NCHAR, as the following example shows:"

Reply:
Thanks Dave, your comments have been incorporated 😁